packages/app/README.md

```diff
@@ -10,15 +10,53 @@ This is the front-end for the Cypress App.
 3. Open chrome (or another browser)
 4. It will show the new Vite powered app 
 
-## How it works
+## How the App works
 
 Cypress has two modes: `run` and `open`. We want run mode to be as light and fast as possible, since this is the mode used to run on CI machines, etc. Open mode is the interactive experience, with the command log, snapshots, selector playground, etc.
 
 - **`open`** mode is driven using GraphQL and urql. It shows the full Cypress app, include the top nav, side nav, spec list, etc. You can change between testing types, check your latest runs on the Cypress dashboard, updating settings, etc.
-- **`run`** mode is does not rely on GraphQL. This is so we can be as performant as possible. It only renders the "runner" part of the UI, which is comprised of the command log and AUT iframe.
+- **`run`** mode is does not rely on GraphQL. This is so we can be as performant as possible. It only renders the "runner" part of the UI, which is comprised of the command log, Spec Runner header, and AUT iframe.
 
 The two modes are composed using the same logic, but have slightly different components. You can see where the differences are in `Runner.vue`(src/pages/Specs/Runner.vue). Notice that `<SpecRunnerOpenMode>` receives a `gql` prop, since it uses GraphQL, and `<SpecRunnerRunMode>` does not.
 
+## Router
+
+[`vite-plugin-pages`](https://github.com/hannoeru/vite-plugin-pages) is used to generate routes based on the page-level Vue components contained in `src/pages`. Route configuration that might typically appear in `router.ts` can be set in a `<route>` block in these page components, for example:
```

Comment:
It might be worth noting that these options (name, meta) directly map to Vue Router's options and link to that? It took me a while to make this connection.

Reply:
> One comment, can we link to Vue Router somewhere? It might not be clear that is the underlying routing mechanism (it is, right?)

We link at the bottom, but I agree it would be more clear to mention up top especially if folks aren't familiar with Vue and Vue Router. Will add an intro to connect the dots between the Vite plugin, the `<route>` block in SFC and Vue Router's options.

Reply:
Changes made!